revealInBlock.dfy(21,10): Error: assertion might not hold
revealInBlock.dfy(31,14): Error: assertion might not hold
revealInBlock.dfy(34,10): Error: assertion might not hold
revealInBlock.dfy(44,10): Error: assertion might not hold
revealInBlock.dfy(57,10): Error: assertion might not hold
revealInBlock.dfy(68,10): Error: assertion might not hold
revealInBlock.dfy(81,14): Error: assertion might not hold
revealInBlock.dfy(84,10): Error: assertion might not hold

Dafny program verifier finished with 22 verified, 8 errors
